.jenkins/test_config.yaml

```diff
@@ -44,8 +44,17 @@ stages:
       - name: gsm8k_small_g2_tp2_mss
         flavor: g2.s
         command: cd .jenkins/lm-eval-harness && bash run-tests.sh -c configs/models-mss.txt -t 2
-  - name: test_gsm8k_spec_decode
-    steps:
       - name: gsm8k_small_g2_tp1_spec_decode
         flavor: g2
-        command: cd .jenkins/lm-eval-harness && bash run-tests.sh -c configs/models-mss.txt -t 1
\ No newline at end of file
+        command: cd .jenkins/lm-eval-harness && bash run-tests.sh -c configs/models-mss.txt -t 1
+   - name: test_gsm8k_spec_decode
```

Comment:
I tried running it in jenkins:
Formatting issue:
hudson.remoting.ProxyException: while parsing a block collection
13:52:38   in 'reader', line 3, column 3:
13:52:38        - name: test_gsm8k_small_models

Reply:
Oh, I found there is extra ' ' at line50. I fixed it by commit [e689218](https://github.com/HabanaAI/vllm-fork/pull/524/commits/e689218e86fd82af6f06d38ac31b832ce4783e8f)
Please check again. 

BTW, is there a way I can test jenkis in case there are still format issues?